packages/clerk-js/src/v4/styledSystem/theme.ts

```diff
@@ -0,0 +1,25 @@
+import { colors } from './colors';
+import { createInternalTheme } from './createInternalTheme';
+import { shadows } from './shadows';
+import { radii, sizes, space } from './sizes';
+import { fonts, fontSizes, fontWeights, letterSpacings, lineHeights } from './typography';
+
+export const baseTheme = createInternalTheme({
```

Comment:
The `createInternalTheme` function accepts a `theme` of type `T`. This generic type is not restricted on purpose: this makes the `baseTheme` strictly typed after the `createInternalTheme` call while still allowing us to add theme tokens without changing types while developing.

It'd be a good idea to narrow down the `T` type once we finalise the tokens for the internal base theme for extra safety